core/kazoo_events/src/kz_hooks_util.erl

```diff
@@ -251,7 +251,10 @@ lookup_account_id(JObj) ->
     case kz_call_event:account_id(JObj) of
         'undefined' ->
             Number = get_inbound_destination(JObj),
-            case kz_cache:peek_local(?HOOKS_CACHE_NAME, cache_key_number(Number)) of
+            case not kz_term:is_empty(Number)
```

Comment:
We're trying not to allow the right-hand side of `andalso` and `orelse` to be non-boolean returns (even though technically allowed).

I'd write this using a function call instead:

```erlang
        Number = get_inbound_destination(JObj),
        lookup_account_id_by_number(Number, kz_term:is_empty(Number));
...

lookup_account_id_by_number(_Number, 'true') -> {'error', 'not_found'};
lookup_account_id_by_number(Number, 'false') ->
    case kz_cache:peek_local(...)
```

Reply:
👍  No problem, it will also remove the nested case so thats also a plus. Il make the change now.